test/template-tests/testcases/deploymentTemplate/Secure-String-Parameters-Cannot-Have-Default.test.ps1

```diff
@@ -0,0 +1,15 @@
+﻿param(
+[Parameter(Mandatory=$true,Position=0)]
+[PSObject]
+$TemplateObject
+)
+foreach ($parameter in $templateObject.parameters) {
+    if ($parameter.Type -eq 'securestring' -and $parameter.defaultValue) {

```

Comment:
can you add a comment to this to note that empty string is not incorrectly flagged by this test (because it's allowed)